api.proto

```diff
@@ -192,6 +192,7 @@ message Ack {
     string    ackInbox         = 5; // NATS subject to publish acks to
     string    correlationId    = 6; // User-supplied value from the message
     AckPolicy ackPolicy        = 7; // The AckPolicy sent on the message
+    int64     timestamp        = 8 [jstype=JS_STRING]; // When the message was committed
```

Comment:
Based on the implementation in https://github.com/liftbridge-io/liftbridge/pull/254, the timestamp being returned is not when the message was committed but rather when it was _received by the server_.